plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -2393,18 +2394,21 @@ else if (rowNumber % 19 == 1) {
 
     @Test
     public void testPartitionSchemaMismatch()
+            throws Exception
```

Comment:
I guess the purpose of the test is to validate the error message when we try to read schema with a type which is not yet supported. Can we either add a new test which validates the error message (by changing it to a different unsupported type) ?

Reply:
Modified the DDL statement of the table `trino_test_partition_schema_change` to make change the column to `float` instead of `double`.